### PR TITLE
Add aiops prod argo

### PR DESCRIPTION
**Pull request by tumido**
## Related Issues

Requires: https://github.com/AICoE/internal-data-hub/pull/8,  https://redhat.service-now.com/surl.do?n=RITM0752436

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Re-add the `aiops-prod-argo` namespace on OCP Prod cluster to be watched. Also cleanup the resource inclusion list to allow watching of Argo Events resources on OCP prod (and remove duplicit entries for PaaS Stage)

## Checklist for migrating an Application to ArgoCD

When migrating an application's deployment to be managed by ArgoCD use the following checklist to verify your process.

Items to complete before making a migration PR
- [x] Ensure your application manifests can be built using Kustomize.
- [x] If using secrets, make sure to include the .sops.yaml file in your repository.
- [x] Make sure to create the role granting access to namespace. See [here](cluster_ns_management.md#deploying-to-a-namespace) for more info. This role is tracked in your application manifests.

Items to include within the migration PR
- [x] Ensure your namespace exists in the cluster spec in the [prod-vars.yaml](../vars/prod-vars.yaml) file (for the appropriate cluster).
- [x] If you are switching between ArgoCD managed namespaces, and that namespace was deleted in OCP, then ensure it is also removed from [prod-vars.yaml](../vars/prod-vars.yaml).
- [x] Ensure the application repository is added in the `argo_cm` file in [prod-vars.yaml](../vars/prod-vars.yaml).
- [x] Ensure that all OCP resources that will be managed by ArgoCD on this cluster are included in the `inclusions` list in [prod-vars.yaml](../vars/prod-vars.yaml). See [here](cluster_ns_management.md#cluster-inclusions) for more info.
- [x] Create the ArgoCD Application, see [here](application_management.md) for instructions.

### CODE REVIEW

**Comment by tumido** [comment]
/hold

until dependencies are merged.

File: manifests/overlays/prod/configs/argo_cm/resource.inclusions
Comment (HumairAK) [inline manifests/overlays/prod/configs/argo_cm/resource.inclusions:74]: Why remove these?

File: manifests/overlays/prod/secrets/clusters/apps.ocp.prod.psi.redhat.com.enc.yaml
Comment (HumairAK) [inline manifests/overlays/prod/secrets/clusters/apps.ocp.prod.psi.redhat.com.enc.yaml:13]: you have to sops edit this file, i.e. 

`$ sops apps.ocp.prod.psi.redhat.com.enc.yaml ` 

Then add the namespace in your editor and save/exit

**Review by HumairAK — APPROVED** [review]

**Review by anishasthana — CHANGES_REQUESTED** [review]
The change to the cluster spec will cause ksops to break as the change was not made via the sops cli.

**Comment by tumido** [comment]
Hmm, bummer. @anishasthana Do you know how I can make the change via sops cli if I don't have the cert to decrypt the file in the first place?

File: manifests/overlays/prod/configs/argo_cm/resource.inclusions
Comment (tumido) [reply manifests/overlays/prod/configs/argo_cm/resource.inclusions:74]: Because they are added by the [line 88](https://github.com/AICoE/aicoe-cd/pull/109/files#diff-9f4c2716bbd7db75239fab641800396cR88)

**Comment by tumido** [comment]
/unhold

The PSI Service Now ticket was resolved. The resources can be watched on the cluster now. This PR is mergeable again.

@anishasthana @HumairAK

**Review by HumairAK — APPROVED** [review]

**Pull request merged**
